### PR TITLE
Switch to glide in favor of submodules for dependency management

### PR DESCRIPTION
**Pull request by cinchurge**
Switch to glide for managing vendor libraries instead of using submodules which are much more painful to manage.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 bin/
 pkg/
 /_output
+vendor
```
Comment (jdef) [inline .gitignore]: nit: `vendor/` please

**Comment by jdef** [comment]
Also, the `next` branch has been merged to master, please submit a PR against `master` instead

**Comment by jdef** [comment]
xref #287

**Comment by pires** [comment]
Seems this becomes obsolete if #287 is to be merged.

**Comment by jdef** [comment]
closing in favor of #287

**Pull request closed without merging**